widgets/blueprintCatalog/src/RepositoryCatalog.tsx

```diff
@@ -33,57 +78,65 @@ const RepositoryCatalog: FunctionComponent<RepositoryViewProps> = ({
                     <Grid className="contentBlock">
                         <Grid.Row className="bottomDivider">
                             <Grid.Column width="16">
-                                <ExternalBlueprintImage url={item.image_url} width={50} />
-                                <Header>
-                                    <a href={item.html_url} target="_blank" rel="noopener noreferrer">
-                                        {item.name}
-                                    </a>
-                                </Header>
+                                <ExternalBlueprintImage url={image_url} width={50} />
+                                <Header>{name}</Header>
                             </Grid.Column>
                         </Grid.Row>
 
-                        <Grid.Column width="16">{item.description}</Grid.Column>
+                        <Grid.Column width="16">{description}</Grid.Column>
 
                         <Grid.Row className="noPadded">
                             <Grid.Column width="4">
-                                <h5 className="ui icon header">{t('properties.created')}</h5>
+                                <h5 className="ui icon header">{t('created')}</h5>
                             </Grid.Column>
-                            <Grid.Column width="12">{item.created_at}</Grid.Column>
+                            <Grid.Column width="12">{created_at}</Grid.Column>
                         </Grid.Row>
 
                         <Grid.Row className="noPadded">
                             <Grid.Column width="4">
-                                <h5 className="ui icon header">{t('properties.updated')}</h5>
+                                <h5 className="ui icon header">{t('updated')}</h5>
+                            </Grid.Column>
+                            <Grid.Column width="12">{updated_at}</Grid.Column>
+                        </Grid.Row>
+                    </Grid>
+                    <Grid container>
+                        <Grid.Row className="noPadded" style={{ marginBottom: '1rem' }}>
+                            <StyledGridColumnButtons width="8">
+                                <StyledLinkButton
+                                    circular
+                                    icon="github"
+                                    onClick={() => Stage.Utils.Url.redirectToPage(html_url)}
+                                />
+
+                                <Button
+                                    circular
+                                    icon="info"
+                                    loading={readmeLoading === name}
+                                    className="readmeButton noPadded"
```

Comment:
As we've removed styling button by `readmeButton` class name, we can safely remove it
The removal has been done here within this file:  `widgets/blueprintCatalog/src/widget.css`

```suggestion
                                    className="noPadded"
```

Reply:
adjusted: https://github.com/cloudify-cosmo/cloudify-stage/commit/a009c1adb29dbe93ef463c1f34db220e57c6f1b2